tests/scripts/localPathPV.sh

```diff
@@ -10,7 +10,12 @@ if [ ! -b "${test_scratch_device}" ] ; then
   exit 1
 fi
 
-lsblk
+# dump data of the disk to check whether it is an osd or not
+sudo dd if="$test_scratch_device" of=data bs=4k count=1
+sudo cat data
```

Comment:
It prints the raw binary data to stdout. It's difficult to read and it would break the terminal's setting when running in the local environment. How about using the binary dump tools like `hexdump` or `xxd`? In addition, I consider 4K is too long and printing "bluestore" signature and fsid is enough.

Reply:
Agreed, this was mostly for quick debug, this commit is going away :)